### PR TITLE
Changed font

### PR DESCRIPTION
**Pull request by ubionexd**
I was annoyed by the default HTML font.

### CODE REVIEW

**Comment by enderger** [comment]
Probably a good idea to set the font, though we probably should use the ones defined [here](https://github.com/modrinth/omorphia/blob/918898d6feeed83d7b6b87302e352006b2b5fee2/src/styles/variables/typography.postcss#L17) for consistency.

File: assets/style.css [hunk 1]
```diff
@@ -21,6 +21,8 @@ body {
     justify-content: center;
     align-items: center;
     text-align: center;
+    font-family: Inter, -apple-system, BlinkMacSystemFont, Segoe UI, Oxygen,
+    Ubuntu, Roboto, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
```
Comment (enderger) [inline assets/style.css:25]: Probably should match [this](https://github.com/modrinth/omorphia/blob/918898d6feeed83d7b6b87302e352006b2b5fee2/src/styles/variables/typography.postcss#L17)

**Review by enderger — CHANGES_REQUESTED** [review]

**Review by enderger — APPROVED** [review]
LGTM

**Comment by enderger** [comment]
Ignore the cancelled CI, probably shouldn't run the publish workflow on PRs :facepalm:.

**Comment by ubionexd** [comment]
Thanks. Deleted the PR's branch.

**Pull request merged**